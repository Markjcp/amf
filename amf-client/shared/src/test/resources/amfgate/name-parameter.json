{
  "swagger": "2.0",
  "info": {
    "title": "Test"
  },
  "paths": {
    "/endpoint": {
      "parameters": [
        {
          "in": "header",
          "type": "integer"
        }
      ],
      "get": {
        "parameters": [
          {
            "in": "query",
            "type": "string"
          }
        ]
      }
    }
  }
}