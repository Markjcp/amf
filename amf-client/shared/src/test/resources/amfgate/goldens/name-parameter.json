{
  "swagger": "2.0",
  "info": {
    "title": "Test",
    "version": "1.0"
  },
  "paths": {
    "/endpoint": {
      "parameters": [
        {
          "in": "header",
          "type": "integer",
          "name": "gen-name-1"
        }
      ],
      "get": {
        "parameters": [
          {
            "in": "query",
            "type": "string",
            "name": "gen-name-2"
          }
        ],
        "responses": {}
      }
    }
  }
}
